### PR TITLE
Travis-ci: Build on Ubuntu Trusty

### PR DESCRIPTION
**Pull request by JBBgameich**
This seems to work already, but most likely the code is not really clean yet. I'll clean it up tomorrow if needed

### CODE REVIEW

File: utils/travis-dependencies.sh [hunk 1]
```diff
@@ -4,27 +4,38 @@ echo "*****************************************"
...
+            qt59tools \
+            qt59x11extras \
+            qt59svg \
+            qt59quickcontrols2 \
+            -y -t trusty-backports
+
+# KF5
+wget -c "https://github.com/JBBgameich/precompiled-kf5-linux/releases/download/KF5.40/kf5.40-gcc6-linux64-release.tar.xz"
+tar xf kf5.40-gcc6-linux64-release.tar.xz
+sudo cp -Rf kf5-release/* /opt/qt*/
```
Comment (LNJ2) [inline utils/travis-dependencies.sh]: very trivial, but why use `-R`, if you can also use `-r`?

File: utils/travis-dependencies.sh [hunk 2]
```diff
@@ -4,27 +4,38 @@ echo "*****************************************"
...
+
+# Gloox
+wget -c "https://github.com/JBBgameich/precompiled-kf5-linux/releases/download/KF5.40/libgloox17_1.0.20-1_amd64.deb"
+wget -c "https://github.com/JBBgameich/precompiled-kf5-linux/releases/download/KF5.40/libgloox-dev_1.0.20-1_amd64.deb"
```
Comment (LNJ2) [inline utils/travis-dependencies.sh]: ... why are the gloox builds in the KF5 repo?

File: utils/travis-dependencies.sh (hunk 1 above)
Comment (JBBgameich) [reply utils/travis-dependencies.sh]: I copied this from an example in the AppImage docs, there's no real reason.

File: utils/travis-dependencies.sh (hunk 2 above)
Comment (JBBgameich) [reply utils/travis-dependencies.sh]: easy :)

**Pull request merged**